modules/user/user.service.js

```diff
@@ -101,17 +107,24 @@ class UserService {
       throw new UserInputError('Errors', { errors });
     }
 
-    const user = await this.getUserByFieldOrThrow('email', email);
+    const user = await this.getUserByFieldOrThrow(
+      'email',
+      email,
+      WRONG_CREDENTIALS[language].value,
```

Comment:
Revert this change, please, we are going to send back unlocalized error messages and localize them only on backend

Reply:
removed